QEfficient/exporter/export_hf_to_cloud_ai_100.py

```diff
@@ -212,12 +220,10 @@ def export_kvstyle_transformed_model_to_onnx(model_name: str,
         multi_query_value = getattr(config, "multi_query")
         if multi_query_value:
             n_heads = 1  # MQA
-            d_head = config.hidden_size // config.num_attention_heads
-            n_layer = 1  # Due to multi query
```

Comment:
can we just change this to `n_layer=config.num_hidden_layers`? for MQA

Reply:
Only n_heads is different in case of MQA which is inside if else condition, rest 2 lines are common so i did this to avoid duplication.